README.md

```diff
@@ -1 +1,154 @@
-# Hello-Microverse-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
```

Comment:
As requested by the previous reviewer it is important that we customize every section of the [readme TEMPLATE](https://github.com/microverseinc/readme-template). to make our readme well-detailed and stand out from a bunch of others. 👌

Kindly add the `Future Features`, including 1-3 potential features to be added. 😊👍